eigenda/proxy.go

```diff
@@ -123,21 +123,22 @@ func (c *client) GetData(ctx context.Context, comm []byte) ([]byte, error) {
 
 	defer resp.Body.Close()
 
-	if resp.StatusCode != http.StatusOK {
-		return nil, fmt.Errorf("received unexpected response code: %d", resp.StatusCode)
+	b, err := io.ReadAll(resp.Body)
+	if err != nil {
+		return nil, err
 	}
 
-	if resp.StatusCode == http.StatusServiceUnavailable {
-		return nil, SvcUnavailableErr
```

Comment:
Did you really mean to remove this? Is ErrSvcUnavailable returned somewhere else?

Reply:
ahh incredible catch 🙏🏻